test/descriptive-link-text.md

```diff
@@ -0,0 +1,23 @@
+# Descriptive link text
+
+[Learn about Javascript](https://example.com/javascript/about)
+
+[Read about Javascript](https://example.com/javascript/about)
+
+Learn about [JavaScript](https://example.com/javascript/about).
```

Comment:
Let's please have an example of every default banned text in this file. Again, it shouldn't matter, but it will alert if someone ever accidentally deletes or corrupts one.